cmd/mattermost-mattermod/main.go

```diff
@@ -71,6 +72,15 @@ func main() {
 		}
 	}()
 
+	c := cron.New()
+
+	_, err = c.AddFunc("0 2 * * *", s.RefreshMembers)
```

Comment:
Should we run this at a different time (before or after) jobserver cron

https://github.com/mattermost/mattermost-mattermod/blob/54f85a29c3cc53a961862ff31f35427c5276e98a/cmd/jobserver/main.go#L69

Reply:
I picked same hours to have a same member list in `mattermod-jobserver` and `mattermod` almost at the same time. I do not know how will the applications behave if they have different set of users.